arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
```

Comment:
I see the you are removing the existing property. This potentially will be a breaking change in the generated SDK. 

Breaking changes are not recommended without updating the api-version.